leveldown.js

```diff
@@ -145,14 +149,22 @@ Multilevel.prototype._clearRequests = function (closing) {
   }
 }
 
+Multilevel.prototype._serializeKey = function (key) {
+  return Buffer.isBuffer(key) ? key : Buffer.from(String(key))
+}
+
+Multilevel.prototype._serializeValue = function (value) {
+  return Buffer.isBuffer(value) ? value : Buffer.from(String(value))
+}
+
 Multilevel.prototype._get = function (key, opts, cb) {
   if (this._db) return this._db._get(key, opts, cb)
 
   var req = {
     tag: 0,
     id: 0,
     key: key,
-    valueEncoding: opts.valueEncoding || (opts.asBuffer === false ? 'utf-8' : 'binary'),
+    valueAsBuffer: opts.asBuffer,
```

Comment:
We no longer have access to `key/ValueEncoding` options here, but we don't need it. We only have to care about returning a buffer or string (which at a higher level, is dictated by `encoding-down` and the user's choice of encoding).